### PR TITLE
[css-fonts] Add font-size: math, math-depth and math-style

### PR DESCRIPTION
**Pull request by fred-wang**
This imports the proposal from MathML Core to manage font scaling in math
formulas using a new 'math-depth' property. This mechanism involves a new
'math-style' property that describes compactness of formulas as well as
a new 'math' keyword to specify relative font-size change using the a
scale factor calculated from the 'math-depth' changes.

https://github.com/w3c/csswg-drafts/issues/5389
https://github.com/w3c/csswg-drafts/issues/5387
https://github.com/w3c/csswg-drafts/issues/5389#issuecomment-694361834
https://github.com/w3c/csswg-drafts/issues/5387#issuecomment-690677637

### CODE REVIEW

**Comment by fred-wang** [comment]
This probably needs improvements, feedback welcome ;-)

Current test/implementation status is:

Tests: css/css-fonts/math-script-level-and-math-style/ (need to be updated and extended) as well as indirect effect on MathML in mathml/

Chrome:
math-style implemented, https://chromium-review.googlesource.com/c/chromium/src/+/2421110
math-level to be re-implemented, https://bugs.chromium.org/p/chromium/issues/detail?id=1023292
font-size: math not implemented yet

Firefox:
exposing internal -moz-math-display, https://bugzilla.mozilla.org/show_bug.cgi?id=1665975
internal -moz-math-level not exposed yet
font-size: math not implemented yet

Safari:
math-style being implemented, https://bugs.webkit.org/show_bug.cgi?id=216702
math-level not implemented yet
font-size: math not implemented yet

**Comment by fantasai** [comment]
Hi @fred-wang ! I think we could use a bit of discussion about where exactly all this goes, but I do suspect it fits better into the MathML Core spec rather than the css-fonts-4 spec. The core processing of this all is fitted to the MathML layout model, and not particularly relevant generally to CSS, is why I'm thinking this.

(Note: for the `display` and `font-size` keywords, if you're using Bikeshed it's possible to use class="propdef partial" to indicate adding to, rather than redefining, the property; I'm not sure what the equivalent syntax would be in ReSpec.)

File: css-fonts-4/Overview.bs [hunk 1]
```diff
@@ -838,7 +838,7 @@ Font size: the 'font-size' property</h3>
 
     <pre class="propdef">
     Name: font-size
-    Value: <<absolute-size>> | <<relative-size>> | <<length-percentage>>
+    Value: <<absolute-size>> | <<relative-size>> | <<length-percentage>> | math
```
Comment (fred-wang) [inline css-fonts-4/Overview.bs:841]: This change is a mistake, it must be reverted

**Comment by fred-wang** [comment]
Closing this per https://github.com/w3c/csswg-drafts/pull/5536#issuecomment-697114884

- math-depth and math-style are in MathML Core.
- font-size: math was added in https://github.com/w3c/csswg-drafts/commit/09efeb0f1948f5aec4afc3672c1ab0c264abab36

**Pull request closed without merging**